src/cgi-bin/connection.lua

```diff
@@ -21,6 +21,13 @@ function configure_network(network, encryption, key)
 	else
 		uci.delete("wireless", "sta", "key")
 	end
+	local file = io.open("/etc/config/MAC_0", "r")
```

Comment:
This has changed due to recent changes. now need to call "generate_mac 0" from "/lib/functions/mac.sh" to get the wlan0 mac address which should be added in the /etc/config/wireless